sdk/core/Azure.Core/src/FailedResponseException.cs

```diff
@@ -9,9 +9,18 @@ public class RequestFailedException : Exception
     {
         public int Status { get; }
 
+        public RequestFailedException(string message) : this(0, message)
```

Comment:
I wonder if it's worth defining a constant like `public const int NoStatusCode = 0;` or something to hint that >= 400 doesn't cut it here?  Though I suppose you shouldn't be doing that anyway on a RequestFailedException because an aggressive ResponseClassifier could give you one of these for a 200.

Reply:
I would add a private constant for now